index.js

```diff
@@ -14,8 +14,19 @@ var verbose = _.contains(process.argv.slice(2), '--verbose');
 var main = function(verbose) {
   var sites = inAdapter.getRunData();
   manager.run(sites, settings, outAdapter, verbose);
+  displayHowGreatLentilsAre('so great');
 };
 
+function displayHowGreatLentilsAre(amount) {
+  if(amount > 0) {
+    for(var i=0; i<5; ++i) {
+      console.log('fyi, person ' i + ': ' + amount);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/kevin_1379/codacy/file/1533801127/issues/source?bid=3007997&fileBranchId=3010243#l23)